api/tests/unit/segments/test_unit_segments_views.py

```diff
@@ -175,11 +176,14 @@ def test_audit_log_created_when_segment_updated(project, segment, client):
 
     # Then
     assert res.status_code == status.HTTP_200_OK
+
+    # Three audit log records created, two for the primary
+    # segment and one for the newly versioned segment.
```

Comment:
Hmm... this feels odd. I don't think we should be creating new audit log records for older versions of segments - the operation of updating a segment should only create a single audit log record, the fact that a versioned segment is created is a byproduct of that operation. 

We can override the `get_skip_audit_log` method on the `Segment` (and maybe `SegmentCondition`) models to ensure we're not writing unnecessary audit log records.

Reply:
Ok this is now done.